docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  postgres:
+    image: postgres:11
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=dp-regel-api
```

Comment:
Wrong POSTGRES_DB name. Should match db name in LOCAL configuration  prop map(dp-inntekt-db)

Reply:
fixed!